docs/advanced/csidriver.md

```diff
@@ -8,23 +8,29 @@ title: "Third-Party Storage Support"
   <link rel="canonical" href="https://docs.harvesterhci.io/v1.4/advanced/csidriver"/>
 </head>
 
-_Available as of v1.2.0_
+_Available as of v1.5.0_
 
-Harvester now offers the capability to install a [Container Storage Interface (CSI)](https://kubernetes-csi.github.io/docs/introduction.html) in your Harvester cluster. This allows you to leverage external storage for the Virtual Machine's non-system data disk, allowing you to use different drivers tailored for specific needs, whether for performance optimization or seamless integration with your existing in-house storage solutions.
+Harvester now supports provisioning of root volumes and data volumes using external [Container Storage Interface (CSI)](https://kubernetes-csi.github.io/docs/introduction.html) drivers. This enhancement allows you to select drivers that meet specific requirements, such as performance optimization or seamless integration with existing internal storage solutions.
 
-:::note
+:::important
+
+The matrix below shows the supported capabilities of the validated CSI drivers.
```

Comment:
Does it make sense to include a href here to the SUSE certified storage for virtualization? E.g., _"For more information on SUSE certified 3rd-party storage, see https://www.suse.com/product-certification/suse-certified/virtualization-certification/."_

Reply:
Thanks @ihcsim,
Actually, I did not know this page. Do we already have the certified storage?